CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Contribute to ResumeWorded
+
+Welcome to ResumeWorded ! We're excited that you're interested in contributing. Please take a moment to read this guide to understand how you can contribute to the project !
+
+## Code of Conduct 
+
+Before you start contributing, We expect all contributors to follow these guidelines to create a respectful and inclusive environment for everyone.
+
+## How to Contribute ?
+
+### Submitting an issue :-
+
+If you feel ,there's any bug or feature improvement Please raise an issue.
+We are here to improve 🙌
+
+### Submitting Pull Request :-
+If you'd like to contribute code or documentation changes, you can submit a pull request (PR) to our repository. Here's how:
+
+1. Fork the repository to your GitHub account.
+   1. Click the fork button on the repository.
+   2. Click Create Fork button below down.
+   3. Copy the ssh clone url.
+2. Clone your forked repository to your local machine.
+   1. Cloning the forking repository using git clone command.
+3. Create a new branch for your changes.
+4. Make your changes and commit them with clear, concise commit messages.
+5. Push your branch to your forked repository.
+6. Open a pull request from your branch to our repository's main branch.
+
+We will review your PR and provide feedback. Once your contribution is approved, it will be merged into the project.
+
+## Development Environment setup
+
+If you plan to contribute code, follow these steps to set up your development environment:
+
+1. Clone the repository.
+2. Start the development of the project.
+
+Thank you for your interest in contributing to ResumeWorded ! We welcome contributions from the community and appreciate your help in making our project better.
+
```

Comment:
Can we add a section for `local deployment`
- Steps on how contributers can setup the repo and run the project locally.
- A brief about the folder structure of the project.

Reply:
> Can we add a section for `local deployment`
> 
> * Steps on how contributers can setup the repo and run the project locally.
> * A brief about the folder structure of the project.

This part is already there in the src folder readme file

Reply:
ok just add about the local exectution steps.
After updating the changes you can request re-review.